lib/ApiRequestor.php

```diff
@@ -9,12 +9,27 @@
  */
 class ApiRequestor
 {
+    /**
+     * @var string|null
+     */
     private $_apiKey;
 
+    /**
+     * @var string
+     */
     private $_apiBase;
 
+    /**
+     * @var HttpClient\CurlClient
```

Comment:
I think this is more like a `ClientInterface` rather than necessarily a `CurlClient`.